fuel/schemes.py

```diff
@@ -1,10 +1,9 @@
 from abc import ABCMeta, abstractmethod
 
 import numpy
-from picklable_itertools import chain, repeat, imap, _iter
+from picklable_itertools import chain, repeat, imap, xrange, iter_
```

Comment:
What does `xrange` from `picklable_itertools` offer over the built-in? Built-in `xrange` is picklable, right?